### PR TITLE
update test data paths

### PR DESCRIPTION
**Pull request by Mark-S-Hill**
## PR checklist

Closes #XXX 

Updated assigned paths.

### CODE REVIEW

File: software/gatk4/variantfiltration/main.nf [hunk 1]
```diff
@@ -20,7 +20,9 @@ process GATK4_VARIANTFILTRATION {
 
     input:
     tuple val(meta), path(vcf)
-    tuple path(fasta), path(fai), path(dict)
+    path(fasta)
+    path(fai)
+    path(dict)
```
Comment (drpatelh) [inline software/gatk4/variantfiltration/main.nf:25]:
```suggestion
    path fasta
    path fai
    path dict
```

File: software/gatk4/variantfiltration/main.nf [hunk 2]
```diff
@@ -20,7 +20,9 @@ process GATK4_VARIANTFILTRATION {
...
 
     output:
     tuple val(meta), path("*filtered.vcf"), emit: vcf
```
Comment (drpatelh) [inline software/gatk4/variantfiltration/main.nf:28]:
```suggestion
    tuple val(meta), path("*.vcf"), emit: vcf
```

File: tests/software/gatk4/variantfiltration/test.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-  command: nextflow run tests/software/gatk4/variantfiltration -entry test_gatk4_variantfiltration
-    -c tests/config/nextflow.config
+- name: gatk4 variantfiltration test_gatk4_variantfiltration
+  command: nextflow run tests/software/gatk4/variantfiltration -entry test_gatk4_variantfiltration -c tests/config/nextflow.config
   tags:
     - gatk4
     - gatk4_variantfiltration
   files:
     - path: output/gatk4/test.filtered.vcf
+      contains: "AC=2;AN=2;DP=1;DP4=0,0,1,0;MQ=60;MQ0F=0;SGB=-0.379885"
```
Comment (drpatelh) [inline tests/software/gatk4/variantfiltration/test.yml:8]: Nice!

File: tests/software/gatk4/variantfiltration/main.nf [hunk 4]
```diff
@@ -2,17 +2,17 @@
 
 nextflow.enable.dsl = 2
 
-test_options = ['args': '--filter-name "test_filter" --filter-expression "MQ0 > 0"']
+test_options = ['args': '--filter-name "test_filter" --filter-expression "MQ0 > 0"', 'suffix': '.filtered']
```
Comment (drpatelh) [inline tests/software/gatk4/variantfiltration/main.nf:5]: 👌🏽

**Comment by drpatelh** [comment]
Couple of changes @Mark-S-Hill ! Looks great otherwise.

File: software/gatk4/variantfiltration/main.nf (hunk 2 above)
Comment (Mark-S-Hill) [reply software/gatk4/variantfiltration/main.nf:28]: just *.vcf won't work for the output right? As the input vcf will be in the workdir too. I can instead do: "*${options.suffix}.vcf". Which would work.

File: software/gatk4/variantfiltration/main.nf (hunk 1 above)
Comment (Mark-S-Hill) [reply software/gatk4/variantfiltration/main.nf]: Updated.

File: software/gatk4/variantfiltration/main.nf [hunk 5]
```diff
@@ -20,10 +20,12 @@ process GATK4_VARIANTFILTRATION {
...
     input:
     tuple val(meta), path(vcf)
-    tuple path(fasta), path(fai), path(dict)
+    path fasta
+    path fai
+    path dict
 
     output:
-    tuple val(meta), path("*filtered.vcf"), emit: vcf
+    tuple val(meta), path("*${options.suffix}.vcf"), emit: vcf
```
Comment (drpatelh) [inline software/gatk4/variantfiltration/main.nf:28]:
```suggestion
    tuple val(meta), path("*.vcf"), emit: vcf
```

Comment (drpatelh) [reply software/gatk4/variantfiltration/main.nf:28]: Nope it should work fine!

File: software/gatk4/variantfiltration/main.nf [hunk 6]
```diff
@@ -20,10 +20,12 @@ process GATK4_VARIANTFILTRATION {
...
     tuple val(meta), path(vcf)
-    tuple path(fasta), path(fai), path(dict)
+    path fasta
+    path fai
+    path dict
 
     output:
-    tuple val(meta), path("*filtered.vcf"), emit: vcf
+    tuple val(meta), path("*.vcf"), emit: vcf
     path "*.version.txt"                  , emit: version
```
Comment (drpatelh) [inline software/gatk4/variantfiltration/main.nf:29]:
```suggestion
    tuple val(meta), path("*.vcf"), emit: vcf
    path "*.version.txt"          , emit: version
```

File: tests/software/gatk4/variantfiltration/main.nf [hunk 7]
```diff
@@ -2,17 +2,17 @@
...
     input = [ [ id:'test' ], // meta map
-              [ file("${launchDir}/tests/data/genomics/sarscov2/illumina/vcf/test.vcf", checkIfExists: true) ] 
-            ]
-    fasta = [ file("tests/data/genomics/sarscov2/genome/genome.fasta", checkIfExists: true),
-              file("tests/data/genomics/sarscov2/genome/genome.fasta.fai", checkIfExists: true),
-              file("tests/data/genomics/sarscov2/genome/genome.dict", checkIfExists: true)
+              [ file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true) ]
             ]
+    fasta = [ file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true) ]
+    fasta_fai = [ file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true) ]
```
Comment (drpatelh) [inline tests/software/gatk4/variantfiltration/main.nf:13]:
```suggestion
    fai = [ file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true) ]
```

File: tests/software/gatk4/variantfiltration/main.nf [hunk 8]
```diff
@@ -2,17 +2,17 @@
...
+    genome_dict = [ file(params.test_data['sarscov2']['genome']['genome_dict'], checkIfExists: true) ]
+
 
-    GATK4_VARIANTFILTRATION ( input, fasta )
+    GATK4_VARIANTFILTRATION ( input, fasta, fasta_fai, genome_dict )
```
Comment (drpatelh) [inline tests/software/gatk4/variantfiltration/main.nf:17]:
```suggestion
    GATK4_VARIANTFILTRATION ( input, fasta, fai, genome_dict )
```

**Comment by drpatelh** [comment]
Thanks @Mark-S-Hill !

**Review by drpatelh — APPROVED** [review]

**Pull request merged**